DESCRIPTION

```diff
@@ -14,14 +14,17 @@ Imports:
     beans,
     bestNormalize,
     bookdown,
+    broom.mixed,
     corrplot,
     corrr,
     DALEX,
     DALEXtra,
     details,
+    DiagrammeR,
+    doMC,
```

Comment:
Please remove the doMC Import. If this package is used in your notes, turn off evaluation of those parts. doMC does not work on Windows, and the bookclubs are meant to be usable on all machines.